HW9/task_1.py

```diff
@@ -0,0 +1,116 @@
+class Apple:
+    ABBREVIATION = 'APL'
+
+    def __init__(self, department_name: str, workers_quantity: int, total_dep_exp : int | float,
+                 productivity_percent_of_department: int | float, salary_of_worker: int | float):
+        self.__department_name = department_name
+        self.__workers_quantity = workers_quantity
+        self.__productivity_percent_of_department = productivity_percent_of_department
+        self.__total_dep_exp = total_dep_exp
+        self.__salary_of_worker = salary_of_worker
+
+    @staticmethod
+    def company_palce_in_top_list():
+        print('Our company is on the 3rd place')
+
+    @classmethod
+    def workers_quantity_predict_in_2_years(cls, department_name: str, workers_quantity: int | float):
+        quantity_predict = workers_quantity * 1.75
+        return cls(department_name, quantity_predict, None, None, None)
```

Comment:
to avoid passing NOne every time, you can make this parameters None by default in init